packages/neon-core/src/tx/components/TransactionAttribute.ts

```diff
@@ -1,139 +1,216 @@
-import { num2hexstring, num2VarInt, StringStream, HexString } from "../../u";
-import { TxAttrUsage } from "./txAttrUsage";
-import { NeonObject } from "../../model";
+import {
+  num2VarInt,
+  StringStream,
+  HexString,
+  NeonSerializable,
+  reverseHex,
+  base642hex,
+} from "../../u";
+import { parseEnum } from "../../internal";
+
+export enum TransactionAttributeType {
+  HighPriority = 0x1,
+  OracleResponse = 0x11,
+}
 
-const maxTransactionAttributeSize = 252;
+export enum OracleResponseCode {
+  // Indicates that the request has been successfully completed.
+  Success = 0x00,
+  // Indicates that the protocol of the request is not supported.
+  ProtocolNotSupported = 0x10,
+  // Indicates that the oracle nodes cannot reach a consensus on the result of the request.
+  ConsensusUnreachable = 0x12,
+  // Indicates that the requested Uri does not exist.
+  NotFound = 0x14,
+  // Indicates that the request was not completed within the specified time.
+  Timeout = 0x16,
+  // Indicates that there is no permission to request the resource.
+  Forbidden = 0x18,
+  // Indicates that the data for the response is too large.
+  ResponseTooLarge = 0x1a,
+  // Indicates that the request failed due to insufficient balance.
+  InsufficientFunds = 0x1c,
+  // Indicates that the content-type of the request is not supported.
+  ContentTypeNotSupported = 0x1f,
+  // Indicates that the request failed due to other errors.
+  Error = 0xff,
+}
 
-export interface TransactionAttributeLike {
-  usage: number | string;
-  data: string | HexString;
+export interface HighPriorityTransactionAttributeJson {
+  type: string;
 }
 
-export interface TransactionAttributeJson {
-  // Name of TransactionAttributeUsage in English
-  usage: string;
-  // Base64-encoded data
-  data: string;
+export interface OracleResponseTransactionAttributeJson {
+  type: string;

```

Comment:
in typescript, you can enforce this to be a static string "OracleResponse"  to be stricter. (Similarly for HighPriority)

Reply:
I think the change is what you asked for